utils/design-vars.stanza

```diff
@@ -14,7 +14,7 @@ public pcb-enum ocdb/design-vars/DensityLevel:
 public var OPERATING-TEMPERATURE = [0.0 25.0]
 
 ; Part selection parameters
-public var OPTIMIZE-FOR = ["area"]
+public var OPTIMIZE-FOR: Tuple<String> = ["area"]
```

Comment:
Type hinting because the type was wrongly inferred to [String] while you should be able to assign a Tuple<String>:
```
tools/explorer.stanza:46.10: Cannot assign expression of type Tuple<String> to variable OPTIMIZE-FOR
with declared type [String].
```